blueprints/app/files/app/index.html

```diff
@@ -1,5 +1,5 @@
 <!DOCTYPE html>
-<html>
+<html lang="en">
```

Comment:
Is english the standard default for this kind of thing? We won't hurt any feelings?